vhost-user-backend/CHANGELOG.md

```diff
@@ -1,12 +1,19 @@
 # Changelog
+
 ## [Unreleased]
 
+### Added
+### Changed
+### Deprecated
+### Fixed
+
+## v0.18.0
+
 ### Added
 - [[#268]](https://github.com/rust-vmm/vhost/pull/268) Add support for `VHOST_USER_GET_SHARED_OBJECT`
 
 ### Changed
-
-### Deprecated
+- [[#295]](https://github.com/rust-vmm/vhost/pull/295) Updated vm-memory to 0.16.2 and vmm-sys-util to 0.14.0
 
 ### Fixed
```

Comment:
I think we can remove this as well, sorry, I didn't see it before.

Reply:
Sorry, I meant the empty "Fixed" section. The dep update is great!

Reply:
fixed